plugin/evm/vmsync/registry_test.go

```diff
@@ -67,12 +80,12 @@ func TestSyncerRegistry_Register(t *testing.T) {
 			expectedCount: 2,
 		},
 		{
-			name: "duplicate name registration",
+			name: "duplicate id registration",
 			registrations: []*mockSyncer{
 				newMockSyncer("Syncer1", nil),
 				newMockSyncer("Syncer1", nil),
 			},
-			expectedError: "syncer with name 'Syncer1' is already registered",
+			expectedError: "syncer with id 'Syncer1' is already registered",
```

Comment:
(fyi) This is exactly what [this post is about](https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html).